lib/project_types/script/layers/infrastructure/push_package_repository.rb

```diff
@@ -4,10 +4,10 @@ module Script
   module Layers
     module Infrastructure
       class PushPackageRepository
-        def create_push_package(script, script_content, schema, compiled_type)
+        def create_push_package(ctx, script, script_content, schema, compiled_type)
```

Comment:
I think it might make sense to omit `ctx` on this method, and instead add it to this class' constructor or use smart properties like script_service.rb. That way if we add more methods later on, we won't have to keep passing along context.

Probably makes sense to do that for our ScriptRepository and similar classes that need this too. What do you think?